fontc_crater/src/ci/html.rs

```diff
@@ -142,7 +142,12 @@ fn make_table_body(runs: &[RunSummary]) -> Markup {
             prev.map(|p| p.stats.diff_perc_including_failures),
             true,
         );
-        let diff_fmt = format!("{:.3}", run.stats.diff_perc_including_failures);
```

Comment:
I don't love the explicit if 100, how about we just truncate after 3 decimal places, e.g. `format!("{:.3}%", (pct * 1000.0).floor() / 1000.0)  // avoid .9995 printing as 100%` 

The results are slightly more conservative but given the nature of the tool that's OK